### PR TITLE
Adding updated schematics for batch 2 and 3

### PR DESCRIPTION
**Pull request by gorski123**
Batches 2 (10 units) and 3 (15 units) shipped with the following versions of hardware:
Amp: 2B
Power: 3B
Preamp: 3A
Preout: 1A
LED: 2A
Controller: 3A

### CODE REVIEW

**Review by Lohrer — COMMENTED** [review]
I made some changes to the changelogs, let me know if they still all look good.

**Comment by linknum23** [comment]
bump

File: hw/sch/power_board/CHANGELOG.md [hunk 1]
```diff
@@ -7,3 +7,15 @@ Initial release
 * TH2/TL2 pulled high via 4.7k resistor.
 * TH1/TL1 pulled low via 4.7k resistor.
 * 9VA DC-DC supply removed, replaced with 9VA isolated supply ITR0305S09.
+
+## Rev 3.B
+* Power good resistors changed for 5V/12V DC-DC power supplies to make LEDs dimmer
+* 9VA DC-DC supply removed, replaced with 5VA and 9VA isolated supplies.
+* Switched all 3.3V circuitry over to analog ground as the preamp board no longer has digital ground.
+* R4 added to reduce PG_12V to 5V to 3.3V logic.
```
Comment (gorski123) [inline hw/sch/power_board/CHANGELOG.md:15]: R4 added to reduce PG_12V and PG_5V to 3.3V logic?

**Review by gorski123 — COMMENTED** [review]
Small grammatical things but otherwise looks good to me.

File: hw/sch/power_board/CHANGELOG.md (hunk 1 above)
Comment (Lohrer) [reply hw/sch/power_board/CHANGELOG.md:15]: Good catch, it was supposed to be "to reduce PG_12V from 5V to 3.3V logic.

**Pull request merged**